src/Snap.cpp

```diff
@@ -288,12 +277,7 @@ SnapResults SnapManager::Snap
 
    if (mSnapToTime) {
       // Find where it would snap time to the grid
-      mConverter.ValueToControls(
```

Comment:
You no longer use mConverter in the general case to define snapping -- and indeed it is excessively complicated for most uses.

But for the weird cases of PAL and/or NTSC frames, conserve old behavior.  Let SnapToFrames include "NumericConverter.h" and define snapping the old way just for the weird case.

Reply:
I have read carefully about the NTSC drop frames. This excessive complexity is only needed for the display purposes. On top of that:

* There is a question to design team about the chosen format. We do not have `23.97` display format in Audacity. (@LWinterberg @Tantacrul FYI)
* We handle `29.97` in a straight forward way. We have a separate format that accounts for NTSC drop frames.
* I don't really know how important is this format for snapping at all.

So should I change this behavior now? I doubt it and I will have to extend the formats of NumericConverter to do so.

Reply:
Maybe an edge case in little used functionality.

It seems the snap functions for the weird cases might be good as separately registered stuff.